pkg/process/checks/process_rt.go

```diff
@@ -50,7 +52,14 @@ func (r *RTProcessCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 	if len(cpuTimes) == 0 {
 		return nil, errEmptyCPUTime
 	}
-	procs, err := getAllProcesses(cfg)
+
+	// if processCheck haven't fetched any PIDs, return early
+	lastPIDs := Process.GetLastPIDs()
+	if len(lastPIDs) == 0 {
```

Comment:
If this is empty, should we try to call `Process.Run()` to fill the PIDs? This is only helpful when doing `process-agent -check rtprocess`.